src/plugins/intel_gpu/src/graph/program.cpp

```diff
@@ -697,6 +697,10 @@ void program::transfer_memory_to_device() {
             auto& mem = data_node.get_attached_memory();
             auto mem_layout = mem.get_layout();
             auto alloc_type = mem.get_allocation_type();
+
+            if (ov::shape_size(mem_layout.get_shape()) == 0)
+                continue;
```

Comment:
When is it needed? I'd expect that constants doesn't have zero dimensions

Reply:
Probably they could have zero dimensions (it's set during Paged Attention transformation):
`pagedattentionextension:PagedAttentionExtension_8748 input tensor: f16:bfyx:0:nopad` <- here is const input

Reply:
@sshlyapn  Which input it is?

@slyalin Do we expect any constant input with `[0]` shape for PA? From the spec I see some scalar inputs, but the expected shape is `[]` for those.

Reply:
It's set for empty alibi input:

https://github.com/openvinotoolkit/openvino/blob/36e7b30b4fc8a3ec9c9112fba72b9c3c9b9c8d24/src/common/transformations/src/transformations/sdpa_to_paged_attention/state_management_pattern.cpp#L328

Reply:
Hmm, spec says:
```
[11]: alibi_slopes​
- Optional​
- shape [num_kv_heads]​
- type float
```

So I suppose `[0]` means that the input is disabled. @slyalin please confirm.